enclave/build.rs

```diff
@@ -0,0 +1,11 @@
+use std::path::Path;
+
+use libbpf_cargo::SkeletonBuilder;
+
+const SRC: &str = "./src/bpf/enclave.bpf.c";
+
+fn main() {
+    let skel = Path::new("./src/bpf/mod.rs");
```

Comment:
Do we care what the CWD is?

Reply:
I don't know. Should I? It seems to work fine regardless whether I call `cargo build` from the main repo dir (`enclave`) or from the subdir (`enclave/enclave`).

Reply:
I believe this is fine. `build.rs` patterns assume you are in the root of the crate, from what I've seen.

Reply:
Nice. Anyway, in the last push, I switched to making a path with `join`